dataprofiler/tests/labelers/test_column_name_model.py

```diff
@@ -136,37 +168,24 @@ def test_save(self, mock_open, *mocks):
         # setup mock
         mock_file = setup_save_mock_open(mock_open)
 
-        # Save and load a Model with custom parameters
-        parameters = {
-            "true_positive_dict": [
-                {"attribute": "ssn", "label": "ssn"},
-                {"attribute": "suffix", "label": "name"},
-                {"attribute": "my_home_address", "label": "address"},
-            ],
-            "false_positive_dict": [
-                {
-                    "attribute": "contract_number",
-                    "label": "ssn",
-                },
-                {
-                    "attribute": "role",
-                    "label": "name",
-                },
-                {
-                    "attribute": "send_address",
-                    "label": "address",
-                },
-            ],
-            "negative_threshold_config": 50,
-            "include_label": True,
-        }
-
-        model = ColumnNameModel(parameters)
+        model = ColumnNameModel(
+            label_mapping=mock_model_parameters, parameters=self.parameters
+        )
 
         model.save_to_disk(".")
-        self.assertDictEqual(
-            parameters,
-            json.loads(mock_file.getvalue()),
+        self.assertEqual(
+            '{"true_positive_dict": [{"attribute": "ssn", "label": "ssn"}, '
+            '{"attribute": "suffix", "label": "name"}, {"attribute": "my_home_address", '
+            '"label": "address"}], "false_positive_dict": [{"attribute": '
+            '"contract_number", "label": "ssn"}, {"attribute": "role", '
+            '"label": "name"}, {"attribute": "send_address", "label": "address"}], '
+            '"negative_threshold_config": 50, "include_label": true}{"true_positive_dict": '
+            '[{"attribute": "ssn", "label": "ssn"}, {"attribute": "suffix", '
+            '"label": "name"}, {"attribute": "my_home_address", "label": "address"}], '
+            '"false_positive_dict": [{"attribute": "contract_number", "label": "ssn"}, '
+            '{"attribute": "role", "label": "name"}, {"attribute": "send_address", "label": '
+            '"address"}], "negative_threshold_config": 50, "include_label": true}',
+            mock_file.getvalue(),
```

Comment:
should this be a variable to make the assert cleaner? Or does it need to be like this?

Reply:
Not a bad idea! For now, just following the same paradigm in the `RegexModel` test suite